apps/api/v2/src/ee/bookings/2024-08-13/controllers/bookings.controller.ts

```diff
@@ -230,16 +229,13 @@ export class BookingsController_2024_08_13 {
     return {
       status: SUCCESS_STATUS,
       data: recordings,
-      message:
-        "This endpoint will require authentication in a future release. Please update your integration to include valid credentials. See https://cal.com/docs/api-reference/v2/introduction#authentication for details.",
     };
```

Comment:
Is this intentional to introduce Authentication with the same version?   
  
This would break existing endpoints unless theyh have updated?